src/ManageCourses.Api/Mapping/CourseMapper.cs

```diff
@@ -12,6 +12,8 @@ namespace GovUk.Education.ManageCourses.Api.Mapping
 {
     public class CourseMapper : ICourseMapper
     {
+        private readonly SubjectMapper subjectMapper = new SubjectMapper();
```

Comment:
Inject this please

Reply:
Why? No dependencies, no polymorphism, so DI could never change behaviour nor simplify the code